lib/pc_reserve.rb

```diff
@@ -85,6 +86,10 @@ def staff_override
     data["pcrUserData1"]
   end
 
+  def patron_status
```

Comment:
I thought we'd settled on "patron_retrieval_status" to make it clear it's an ETL concern and not about the patron's card status.